lib/puppet/provider/templates/webadministration/_setwebsite.ps1.erb

```diff
@@ -0,0 +1,24 @@
+$resource = @{
+  name            = '<%= "#{resource[:name]}" %>'
+  ensure          = '<%= "#{resource[:ensure]}" %>'
+}
+
+function Try-SetItemProperty
```

Comment:
I would rather see this function pulled out of _newwebsite.ps1.erb into a general erb rather than duplicate it here